### PR TITLE
add function selectEx and test

### PR DESCRIPTION
**Pull request by teethfairy**
add function selectEx and related test

### CODE REVIEW

File: test/mysql.js [hunk 1]
```diff
@@ -70,7 +80,7 @@ it('where in select', function(done) {
     conn.select('table1', 'b', {a: 2}, next)
   },
   function(rows, info, next) {
-    conn.select('table1', ['a','id'], {a: 2}, next)
+    conn.select('table1', ['a','id', 'b'], {a: 2}, next)
```
Comment (yinrong) [inline test/mysql.js]: why change this?

File: README.md [hunk 2]
```diff
@@ -148,7 +148,8 @@ const InterfaceTwo = {
 
 ### method
 ```javascript
-conn.queryReadonly(
+conn.selectEx(
+  /* table */ 'table',
```
Comment (yinrong) [inline README.md]: tableN is better. table is a keyword

File: test/mysql.js [hunk 3]
```diff
@@ -32,8 +32,18 @@ it('table routing', function(done) {
       assert(err.toString().match(/not allowed/))
       next(null, null, null)
     })
+  },/*
```
Comment (yinrong) [inline test/mysql.js]: why comment??

File: test/mysql.js [hunk 4]
```diff
@@ -32,8 +32,18 @@ it('table routing', function(done) {
...
+  function(rows, info, next) {
+    conn.selectEx('table1', 'select * from table1', next)
   },
   function(rows, info, next) {
+    assert(rows.length > 0)
+    conn.selectEx('table', 'update table set a=1', function(err) {
```
Comment (yinrong) [inline test/mysql.js]: why add this test case?  when you remove, be sure you know the reason.

File: test/mysql.js (hunk 1 above)
Comment (teethfairy) [inline test/mysql.js]: 报错，需要获取b属性值

**Comment by yinrong** [comment]
merged by https://github.com/yinrong/node-sqlx/pull/14

**Pull request closed without merging**